apps/web/vibes/soul/primitives/products-carousel/index.tsx

```diff
@@ -20,75 +20,87 @@ export type CarouselProduct = CardProduct;
 interface Props {
   products: Streamable<CarouselProduct[]>;
   className?: string;
-  emptyStateMessage?: string;
+  emptyStateTitle?: string;
+  emptyStateDescription?: string;
```

Comment:
We've been using the pairs title/subtitle for situations like this. Can we rename `emptyStateDescription` to `emptyStateSubtitle`?